src/lemonade_server/server_models.json

```
@@ -318,5 +318,11 @@
         "recipe": "llamacpp",
         "suggested": true,
         "labels": ["hot", "reasoning"]
+    },
+    "GLM-4.5-Air-UD-Q4K-XL-GGUF": {
+        "checkpoint": "unsloth/GLM-4.5-Air-GGUF:UD-Q4_K_XL",
+        "recipe": "llamacpp",
+        "suggested": false,
```

```suggestion
        "suggested": true,
```

Any reason not to suggest it? That just determines whether it shows up in the docs and webapp, or not.